core/test/unit/data/schema/fixtures/utils_spec.js

```diff
@@ -38,63 +38,39 @@ describe('Migration Fixture Utils', function () {
 
         it('should match undefined with no args', function () {
             matchFunc()({get: getStub}).should.be.true();
-            getStub.calledOnce.should.be.true();
```

Comment:
Removed this checks because they are irrelevant to what we're testing (input == correct output)